src/pipeline_v3/core/postgres_base_rls.py

```
@@ -0,0 +1,379 @@
+
+        try:
+            self._async_pool = await asyncpg.create_pool(
+                self._async_dsn,
+                min_size=self.settings.min_connections,
+                max_size=self.settings.max_connections,
+                timeout=30,
+                command_timeout=self.settings.statement_timeout / 1000,  # Convert to seconds
+            )
+            logger.info(f"Initialized async PostgreSQL pool for schema: {self.schema}")
+        except Exception as e:
+            logger.error(f"Failed to initialize async pool: {e}")
```

Ruff (0.11.9)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)